typesafe-builder-checker/src/main/java/org/checkerframework/checker/builder/TypesafeBuilderAnnotatedTypeFactory.java

```diff
@@ -554,6 +554,15 @@ private boolean hasAnnotation(Element element, Class<? extends Annotation> annot
     return element.getAnnotation(annotClass) != null;
   }
 
+  private boolean hasAnnotation(Element element, String annotName) {
+    return element.getAnnotationMirrors().stream()
```

Comment:
Can we use `AnnotationUtils.areSameByName` like in https://github.com/msridhar/returnsrecv-checker/pull/12?

Reply:
Definitely. I guess I need more coffee this morning...

Reply:
😆I only knew about that routine by chance